### PR TITLE
Add support for "markdown" format

### PR DESCRIPTION
**Pull request by lexmag**
Addresses #28.

### CODE REVIEW

File: lib/mix/tasks/bench.ex [hunk 1]
```diff
@@ -19,9 +19,10 @@ defmodule Mix.Tasks.Bench do
 
   ## Options
 
-      -n, --no-pretty
-          Instead of pretty-printing the output, print it in the format that
-          can be parsed by bench.cmp and bench.graph.
+      -f, --format
+          Print it in the specific format.
+
+          One of: raw, plain (default).
```
Comment (alco) [inline lib/mix/tasks/bench.ex]: Markdown is missing.

**Comment by alco** [comment]
Thanks! The code looks good so far, I'll test it later today.

Were you planning to also update the `--format` option for `bench.cmp` in this PR? It's OK to leave it for another one, I'm asking just in case.

File: lib/mix/tasks/bench.ex (hunk 1 above)
Comment (lexmag) [inline lib/mix/tasks/bench.ex]: Oh. :pensive:

**Comment by lexmag** [comment]
Yes, I'd like to put `bench.cmp --format` change into another PR.

**Comment by alco** [comment]
Sorry, I forgot to get back to this. Going to have a closer look today and finally merge it.

**Comment by alco** [comment]
Looks good. Thanks.

One wish for the future: always put an empty line after the first line in the commit message. Otherwise, the output of `git log --oneline` becomes scary:

screen shot 2015-06-03 at 22 38 29

**Comment by lexmag** [comment]
Thanks, will add empty line next time.

**Pull request merged**